### DISCUSSION TITLE
Place third-party scripts inside a <div>

### DISCUSSION THREAD

**Discussion by surjithctly** [vercel/next.js#35428] [Help] [answered]
I'm implementing a 3rd party script where they insert some content on the parent div where the script is placed. 

However, using Next.js, all scripts added via `<Script>` was moved to `head` or `body` which causes the content to be run in the wrong place. I cannot use a normal `script` because of [this error](https://nextjs.org/docs/messages/no-sync-scripts)

I wanted to run a script in an exact place inside a component. eg: `<div> script here </div>`

Here's the [code / live preview
](https://stackblitz.com/edit/nextjs-baga1f?file=pages/index.js)
Any help?

**Comment by aayanqazi** [comment]
Did you try to use it ?
https://nextjs.org/docs/basic-features/script#lazyonload

**Reply by surjithctly** [reply]
Even it's lazyloaded, the script moves to the head, I need to place it where I placed it in the code.

**Comment by surjithctly** [comment, marked as answer]
Using React's `dangerouslySetInnerHTML` works!

```jsx
 <div id="script" dangerouslySetInnerHTML={{ __html: `<script src="..."></script>` }} />
```
